src/phases/move-effect-phase.ts

```diff
@@ -316,6 +321,44 @@ export class MoveEffectPhase extends HitCheckPhase {
     }
   }
 
+  /**
+   * Applies all effects for moves that target one or both sides of the field.
+   * This assumes such effects are implemented with {@linkcode MoveEffectTrigger | POST_APPLY}
+   * effect triggers, and will try to play an animation even if no active Pokemon
+   * are affected.
+   * @param user the {@linkcode Pokemon} using the move
+   */
+  private applyFieldMoveEffects(user: Pokemon): void {
```

Comment:
```suggestion
  private applyFieldMoveEffects(user: Pokemon): void {
    // Lapse `MOVE_EFFECT` effects (i.e. semi-invulnerability) when applicable
    user.lapseTags(BattlerTagLapseType.MOVE_EFFECT);

```
(Just in case we add any `MOVE_EFFECT` battler tags that aren't semi-invulnerability later on)